### ISSUE TITLE
Blank screen While editing default tax rate of a product in Dolibarr V 20.0.2

### ISSUE THREAD

**Issue by sribalajitvm** [Dolibarr/dolibarr#32418]
### Bug

 We are receiving blank screen when we try to edit default tax rate of the product. We are using Second Tax and Third Tax (for INDIA). Screenshot attached is after enabling error display option in PHP setting

### Dolibarr Version

20.0.2

### Environment PHP

8.1

### Attached files

![erro](https://github.com/user-attachments/assets/3dae6291-5a92-4948-b892-6ab63b4dc9aa)

**Comment by erikvanberkum** [comment]
I cannot reproduce with 20.0.2 amd 8.1 can you provide more information? I tested with a single tax.

**Comment by sribalajitvm** [comment]
Yes I can provide more information Please let me know Regards Venkatesan Sent from my iPhoneOn 22 Dec 2024, at 2:55 PM, Erik van Berkum ***@***.***> wrote:﻿
I cannot reproduce with 20.0.2 amd 8.1 can you provide more information?
